glide.yaml

```diff
@@ -6,10 +6,11 @@ import:
 - package: github.com/spf13/viper
 - package: github.com/pkg/errors
 - package: k8s.io/client-go
-  version: ^2.0.0
+  version: 3.0.0
   subpackages:
   - pkg/api/v1
-  - pkg/runtime
   - pkg/apis
+- package: k8s.io/apimachinery
+  version: release-1.6
```

Comment:
> umm, @kadel glide update will modify glide.lock, not glide.yml, right?

yes, only think that automatically modifies `glide.yaml` as far as I know is `glide get`